app/src/components/queue.js

```diff
@@ -0,0 +1,60 @@
+const Bull = require('bull');
+const config = require('config');
+const log = require('npmlog');
+const uuidv4 = require('uuid/v4');
+
+const email = require('./email');
+
+const chesQueue = new Bull('ches', {
+  redis: {
+    host: config.get('redis.host'),
+    password: config.get('redis.password')
+  }
+});
+
+chesQueue.process(async job => {
+  log.info('queue', `Job ${job.id} is processing...`);
+
+  try {
+    if (job.data.message) {
+      const result = await email.sendMailSmtp(job.data.message);
+      job.log(JSON.stringify(result));
+      return result;
```

Comment:
what are we doing with the result?  or is that next sprint?

Reply:
The results instead of being passed back directly to the API caller in the response is now being stored as the end result of the job queue. In a future PR, I am intending for there to be a new GET endpoint that will expose the result via job queue lookup.

Reply:
Refer to #11 for upcoming proposed spec